assets/model_monitoring/components/tests/e2e/test_model_data_collector_preprocessor_e2e.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Thoughts on adding the input MDC preprocessor + data joiner as part of the feature attribution drift signal e2e test?

Reply:
Oh yes we should absolutely do it!! I was thinking we will do it in GSQ but I totally forgot we have FAD. Let me add an item

Reply:
https://msdata.visualstudio.com/Vienna/_workitems/edit/2612466